extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php

```diff
@@ -4,162 +4,164 @@
 		<img src="<?= $user['avatar']; ?>" itemprop="image" class="avatar">
 
 		<div class="avatar-controls">
-			<? if( $canEditProfile ): ?>
+			<? if ( $canEditProfile ): ?>
 				<span>
-					<img src="<?= $wgBlankImgUrl ?>" class="sprite edit-pencil"> <a href="#" id="userAvatarEdit"><?= wfMsg('user-identity-box-edit-avatar'); ?></a>
+					<img src="<?= $wgBlankImgUrl ?>" class="sprite edit-pencil">
+					<a href="#" id="userAvatarEdit"><?= wfMessage( 'user-identity-box-edit-avatar' )->escaped(); ?></a>
 				</span>
 			<? endif; ?>
-			<?php if( $canRemoveAvatar ): ?>
+			<?php if ( $canRemoveAvatar ): ?>
 				<span>
-					<img src="<?= $wgBlankImgUrl ?>" class="sprite trash"> <a id="UserAvatarRemove" data-name="<?= $user['name']; ?>"  href="#" data-confirm="<?= htmlspecialchars(wfMsg('user-identity-remove-confirmation')); ?>"><?= wfMsg('user-identity-box-delete-avatar'); ?></a>
+					<img src="<?= $wgBlankImgUrl ?>" class="sprite trash">
+					<a id="UserAvatarRemove" data-name="<?= $user['name']; ?>" href="#" data-confirm="<?= wfMessage( 'user-identity-remove-confirmation' )->escaped(); ?>">
+						<?= wfMessage( 'user-identity-box-delete-avatar' )->escaped(); ?>
+					</a>
 				</span>
 			<?php endif; ?>
 		</div>
 	</div>
-	
+
 	<div class="masthead-info">
 		<hgroup>
-			<? if( !empty($user['realName']) ): ?>
-				<h1 itemprop="name"><?= htmlspecialchars($user['name']); ?></h1>
-				<h2><?= wfMsg('user-identity-box-aka-label', array('$1' => $user['realName']) ); ?></h2>
-			<? else: ?>
-				<h1 itemprop="name"><?= htmlspecialchars($user['name']); ?></h1>
+			<h1 itemprop="name"><?= htmlspecialchars( $user['name'] ); ?></h1>
+			<? if ( !empty( $user['realName'] ) ): ?>
+				<h2><?= wfMessage( 'user-identity-box-aka-label', $user['realName'] )->plain(); ?></h2>
```

Comment:
We can't escape messages contining HTML. This will available when we finish: https://wikia-inc.atlassian.net/browse/SOC-417